packages/graphql/src/schema/query.ts

```diff
@@ -353,7 +385,8 @@ export const Query = builder.queryType({
           return ParticipantGroupService.getGroupActivityDetails(args, ctx)
         },
       }),
-      getBookmarkedQuestions: t.field({
+
+      getBookmarkedQuestions: asParticipant.field({
```

Comment:
Changed from no authentication at all to `asParticipant` as this query is only used in `frontend-pwa`

Reply:
TODO: ensure that the access rights here are consistent with the new working implementation on the stacked learning element branch

Reply:
Permission settings from the stacked learning element branch were not overwritten during merge, so they should be correct as tested there.